### PR TITLE
i just created a normal project based folders and file structure

### PR DESCRIPTION
**Pull request by developerbello**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Hello, There</title>
```
Comment (abdulfataiaka) [inline public/index.html]: This should be the title for the page

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+</head>
+<body>
+    <section>
+        <span>Hello, Universe</span>
+    </section>
+</body>
+</html>
```
Comment (abdulfataiaka) [inline public/index.html]: Add a new line at the end

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```
Comment (abdulfataiaka) [inline public/index.html]: What is the purpose of this meta tag?

File: template/index.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Hello, There</title>
+</head>
+<body>
+    <section>
+        <span>Hello, World</span>
+    </section>
+</body>
+</html>
```
Comment (abdulfataiaka) [inline template/index.html]: Same comments as those from `public.index.html`.

File: public/index.html (hunk 3 above)
Comment (developerbello) [reply public/index.html]: It is mostly used at head section to define some attributes like the viewport etc. AND also helps the website In the aspect of SEO

File: public/index.html (hunk 1 above)
Comment (developerbello) [reply public/index.html]: done...

File: template/index.html (hunk 4 above)
Comment (developerbello) [reply template/index.html]: done...

**Pull request merged**